<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settingsDialog</class>
 <widget class="QDialog" name="settingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>634</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QListWidget" name="listWidget">
       <property name="maximumSize">
        <size>
         <width>125</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="showDropIndicator" stdset="0">
        <bool>false</bool>
       </property>
       <property name="dragDropMode">
        <enum>QAbstractItemView::NoDragDrop</enum>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>48</height>
        </size>
       </property>
       <property name="movement">
        <enum>QListView::Static</enum>
       </property>
       <property name="flow">
        <enum>QListView::LeftToRight</enum>
       </property>
       <property name="isWrapping" stdset="0">
        <bool>true</bool>
       </property>
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="gridSize">
        <size>
         <width>117</width>
         <height>70</height>
        </size>
       </property>
       <property name="viewMode">
        <enum>QListView::IconMode</enum>
       </property>
       <item>
        <property name="text">
         <string>Channels</string>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/icons/images/channels_icon.png</normalon>
         </iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Jack</string>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/icons/images/jack-icon.png</normalon>
         </iconset>
        </property>
        <property name="flags">
         <set>ItemIsSelectable|ItemIsEnabled</set>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Metadata</string>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/icons/images/text-sgml.png</normalon>
         </iconset>
        </property>
        <property name="flags">
         <set>ItemIsSelectable|ItemIsEnabled</set>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SongDB</string>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/icons/images/server-database.png</normalon>
         </iconset>
        </property>
        <property name="flags">
         <set>ItemIsSelectable|ItemIsEnabled</set>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Remote Control</string>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/icons/images/esd.png</normalon>
         </iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Misc</string>
        </property>
        <property name="icon">
         <iconset>
          <normalon>:/icons/images/applications-development.png</normalon>
         </iconset>
        </property>
        <property name="flags">
         <set>ItemIsSelectable|ItemIsEnabled</set>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="channels">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0" colspan="2">
          <widget class="QListWidget" name="channelList">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="3">
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="4">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="channel_add">
             <property name="text">
              <string>&amp;Add</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="channel_remove">
             <property name="text">
              <string>&amp;Remove</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QToolButton" name="upButton">
             <property name="text">
              <string>&amp;up</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="downButton">
             <property name="text">
              <string>&amp;down</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="2">
          <widget class="QToolBox" name="channelSettings">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>4</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="currentIndex">
            <number>2</number>
           </property>
           <widget class="QWidget" name="general">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>116</width>
              <height>64</height>
             </rect>
            </property>
            <attribute name="label">
             <string>General</string>
            </attribute>
            <layout class="QFormLayout" name="formLayout">
             <item row="1" column="1">
              <widget class="QLineEdit" name="name"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="name_label">
               <property name="text">
                <string>Name:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="color_label">
               <property name="text">
                <string>Color:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QToolButton" name="colorChooser">
               <property name="minimumSize">
                <size>
                 <width>64</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="automation">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>100</width>
              <height>30</height>
             </rect>
            </property>
            <attribute name="label">
             <string>Automation</string>
            </attribute>
           </widget>
           <widget class="QWidget" name="fileplayer_settings">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>309</width>
              <height>256</height>
             </rect>
            </property>
            <attribute name="label">
             <string>Fileplayer Settings</string>
            </attribute>
            <layout class="QFormLayout" name="formLayout_2">
             <item row="0" column="0">
              <widget class="QCheckBox" name="fileplayer_autoload">
               <property name="text">
                <string>enable autoload from playlist</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="alsa_settings">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>100</width>
              <height>30</height>
             </rect>
            </property>
            <attribute name="label">
             <string>ALSA Settings</string>
            </attribute>
           </widget>
           <widget class="QWidget" name="jack_settings">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>100</width>
              <height>30</height>
             </rect>
            </property>
            <attribute name="label">
             <string>JACK Settings</string>
            </attribute>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="jack">
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>40</y>
           <width>421</width>
           <height>27</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Jack Client Name</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit"/>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="metadata">
        <widget class="QCheckBox" name="checkBox">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>30</y>
           <width>311</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Generate Metadata for Stream</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="songdb"/>
       <widget class="QWidget" name="misc"/>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::Reset</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>settingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>418</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>settingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>424</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listWidget</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>243</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
